docs/intro/reusable-apps.txt

```diff
@@ -191,14 +191,14 @@ this. For a small app like polls, this process isn't too difficult.
    .. code-block:: python
        :caption: django-polls/setup.py
 
-       import os
+       from pathlib import Path
        from setuptools import find_packages, setup
 
-       with open(os.path.join(os.path.dirname(__file__), 'README.rst')) as readme:
+       with (Path(__file__).resolve().parent / 'README.rst').open() as readme:
```

Comment:
Is the `.resolve()` necessary? (Also, I'd rather we change this to use `setup.cfg` which is much more straightforward...)

Reply:
I agree. Moved to `setup.cfg` in #12006.